odoo/modules/neutralize.py

```diff
@@ -19,3 +21,19 @@ def get_neutralization_queries(modules):
         if filename:
             with odoo.tools.misc.file_open(filename) as file:
                 yield file.read().strip()
+
+def neutralize_database(cursor, to_stdout=False):
+    installed_modules = get_installed_modules(cursor)
+    queries = get_neutralization_queries(installed_modules)
+    if to_stdout:
+        # pylint: disable=bad-builtin
+        print('BEGIN;')
+        for query in queries:
+            # pylint: disable=bad-builtin
+            print(query.rstrip(";") + ";")
+        # pylint: disable=bad-builtin
+        print("COMMIT;")
```

Comment:
Those bits are only useful for the neutralize CLI, I think they should be moved over there

This function should only do
```py
installed_modules = get_installed_modules(cursor)
queries = get_neutralization_queries(installed_modules)
for query in queries:
    cursor.execute(query)
_logger.info("Neutralization finished")
```
IMO it is ok if we duplicate `installed_modules = get_installed_modules(cursor); queries = get_neutralization_queries(installed_modules)` in the CLI